indexd/blueprint.py

```diff
@@ -86,8 +85,8 @@ def dist_get_record(record):
                 fetcher_client = DOSClient(baseurl=indexd['host'])
                 res = fetcher_client.get(record)
             else:
-                fetcher_client = IndexClient(baseurl=indexd['host'])
-                res = fetcher_client.global_get(record, no_dist=True)
+                res = requests.get(indexd['host'] + '/' + record)
+                handle_error(res)
```

Comment:
👍 This seems like a decent fix for the circular dependency.  One critical change:  Add the `no_dist` parameter to the request.  You can see how indexclient does it:
https://github.com/NCI-GDC/indexclient/blob/master/indexclient/client.py#L63

This will prevent misconfigured indexds from having an infinite loop of distributed requests. 💥

Reply:
🙈 updated!